app/models/presenter/bid.rb

```diff
@@ -9,7 +9,11 @@ def bidder_duns_number
     end
 
     def bidder_name
-      bidder.name || null
+      if Presenter::Auction.new(auction).available?
+        '[Name witheld until the auction ends]'
+      else
+        bidder.name || null
```

Comment:
Good find. @baccigalupi, I think you wrote this ... any problems with `null` -> `nil`?